### PR TITLE
[SQL Server DevOps auditing policy] Support SQL server DevOps audit policy commands

### PR DESCRIPTION
**Pull request by Oleza1972**
Description:
In this PR the following commands have been introduced to support SQL server DevOps policy :

az sql server ms-support audit-policy show

az sql server ms-support audit-policy update

Examples of commands:

sql server ms-support audit-policy update -g my_resource_group -s my_server -n my_database --state Enabled --blob-storage-target-state Enabled --storage-endpoint https://mystorageaccount.blob.core.windows.net/

sql server ms-support audit-policy update -g my_resource_group -s my_server -n my_database --state Enabled --blob-storage-target-state Disabled

sql server ms-support audit-policy update -g my_resource_group -n my_server --state Enabled --log-analytics-target-state Enabled --log-analytics-workspace-resource-id /subscriptions/1111-2222-3333-444-5555555/resourcegroups/my_resource_group/providers/microsoft.operationalinsights/workspaces/my_workspace

sql server ms-support audit-policy update -g my_resource_group -n my_server --state Enabled --log-analytics-target-state Disabled --event-hub-target-state Enabled --event-hub-authorization-rule-id /subscriptions/1111-2222-3333-4444/resourceGroups/my_resource_group/providers/Microsoft.EventHub/namespaces/my_eventhub_namespace/authorizationrules/my_eventhub_auth_rule_name --event-hub my_event_hub_name

Testing Guide

Unit test test_sql_server_ms_support_mgmt covers the test for the change.

History Notes

[SQL] az sql server ms-support audit-policy show: show server's DevOps audit policy
[SQL] az sql server ms-support audit-policy update: update server's DevOps audit policy
[SQL] az sql server ms-support audit-policy wait: place the CLI in a waiting state until a condition of the server's DevOps audit policy is met.

### CODE REVIEW

**Comment by yonzhan** [comment]
sql

File: src/azure-cli/azure/cli/command_modules/sql/commands.py [hunk 1]
```diff
@@ -270,6 +271,23 @@ def load_command_table(self, _):
         g.generic_update_command('update', custom_func_name='server_audit_policy_update', supports_no_wait=True)
         g.wait_command('wait')
 
+    server_dev_ops_audit_settings_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#ServerDevOpsAuditSettingsOperations.{}',
+        client_factory=get_sql_server_dev_ops_audit_settings_operations)
+
+    with self.command_group('sql server ms-support audit-policy',
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/sql/commands.py:278]: would you mind adding `is_preview=True` for your new command group if it is not ready to GA?

Comment (Oleza1972) [reply src/azure-cli/azure/cli/command_modules/sql/commands.py:278]: This feature is ready for GA which will be released in 4-5 weeks from now.
Do you think I still need to set 'is_preview=True'?

Comment (Juliehzl) [reply src/azure-cli/azure/cli/command_modules/sql/commands.py:278]: If there is no changes for your server side and CLI interface, you could not add it.

**Review by Juliehzl — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/sql/commands.py (hunk 1 above)
Comment (Oleza1972) [reply src/azure-cli/azure/cli/command_modules/sql/commands.py:278]: Thank you

**Pull request merged**